src/views/AccessDenied/style.js

```diff
@@ -12,7 +12,7 @@ const useStyles = makeStyles((theme) => ({
     height: '100%',
   },
   quoteContainer: {
-    [theme.breakpoints.down('md')]: {
+    [theme.breakpoints.down('xl')]: {
```

Comment:
Not sure if related to this change, but the error message in _AccessDenied_ page is slightly off (even more than it already was).

Reply:
It's an automatic change made by the script. The graphic change seems to be very light.